content/applications/general/users/access_rights.rst

```diff
@@ -2,66 +2,148 @@
+   :alt: In developer mode, go to the settings page and click Users and Companies, then Groups.
+
+To create a new group, click :guilabel:`New`, select an application, and complete the New Group form
+(detailed below). To modify existing groups, click on a group and edit the contents of the form.
 
-| As a second layer of editing and visibility rules, *Record Rules* can be formed. They overwrite,
-  or refine, the *Access Rights*.
-| A record rule is written using a *Domain*. Domains are conditions used to filter or searching
-  data. Therefore, a domain expression is a list of conditions. For each rule, choose among the
-  following options: *Read*, *Write*, *Create* and *Delete* values.
+.. important::
+   Always test the settings being changed to ensure they are being applied to the correct users.
 
-.. image:: access_rights/groups-record-rules.png
+The group form contains multiple tabs for managing all elements of the group. In each tab, click
+:guilabel:`Add a line` to add a new row for users or rules, and click the :guilabel:`X` icon to
+remove a row.
+
+.. image:: access_rights/groups-form.png
    :align: center
-   :alt: View of a group’s form emphasizing the tab record rules in Odoo
+   :alt: Click each tab in the groups form to modify the settings of the group.
+
+- :guilabel:`Users` tab: lists the current users in the group, users listed in black have
+  administrative rights. Users without administrative access appear in blue.
+- :guilabel:`Inherited` tab: inherited means that users added to this group are automatically added
+  to the groups listed on this tab.
+
+  .. example::
+     For example: If the group *Sales/Administrator* lists the group *Website/Restricted Editor* in
+     its Inherited tab, then any users added to the *Sales/Administrator* group automatically
+     receive access to the *Website/Restricted Editor* group as well.
+
+- :guilabel:`Menus` tab: defines which menus/models the group can have access to.
+- :guilabel:`Views` tab: lists which views in Odoo the group has access too.
+- :guilabel:`Access Rights` tab: lists the first level of rights (models) that this group has access
+  rights to. In this tab, the :guilabel:`Model` column represents the common name of the menu/model,
+  and the :guilabel:`Name` column represents the technical name given to the model. For each model,
+  enable the following options as appropriate:
+
+  - :guilabel:`Read`: users can see the object's existing values.
+  - :guilabel:`Write`: users can edit the object's existing values.
+  - :guilabel:`Create`: users can create new values for the object.
+  - :guilabel:`Delete`: users can delete values for the object.
+
+  .. tip::
+     First try searching for the common name of the model in the drop-down menu of the
+     :guilabel:`Model` column. The :guilabel:`Model` technical name can be found by expanding the
+     model common name by clicking the :guilabel:`expand` button.
```

Comment:
```suggestion
     model common name by clicking the :guilabel:`expand` button, which is represented by a :guilabel:`arrow out of a box` icon.
```
is there an emoji we could use here? I know it's a fairly unique icon, so perhaps not -- but it could be worth looking into...